{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/vue/components/images/LazyImage.css

```diff
@@ -0,0 +1,69 @@
+/*
+|--------------------------------------------------------------------------
+| Image
+|--------------------------------------------------------------------------
+| @namespace: img-
+|
+*/
+.img-Image {
+  position: relative;
+
+  margin: 0;
+
+  background-color: #0e0e0e;
+  overflow: hidden;
+}
+
+.img-Image_Media {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  width: 100%;
+  height: 100%;
+}
+
+.img-Image_Image {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  width: 100%;
+  height: 100%;
+
+  opacity: 0;
+
+  transition: opacity 0.5s linear;
+
+  @supports (object-fit: cover) {
+    object-fit: cover;
+  }
```

Comment:
The `object-fit` stuff is helpful, the `@supports` is probably not needed now tbf because of the `img-Image_Image-ie` below.

Even if the image is absolutely positioned it doesn't guarantee the image will be scaled properly, `object-fit: cover` basically make the image work like `background-size: cover` so if the image is actually bigger than the image container it's given it will pull itself off the edges to maintain it's size.